gma/integration_test/src/integration_test.cc

```diff
@@ -987,6 +987,44 @@ TEST_F(FirebaseGmaTest, TestAdViewLoadAdDestroyNotCalled) {
   WaitForCompletion(ad_view->LoadAd(GetAdRequest()), "LoadAd");
   delete ad_view;
 }
+
+TEST_F(FirebaseGmaTest, TestAdViewAdSizeCompareOp) {
+  using firebase::gma::AdSize;
+  EXPECT_TRUE(AdSize(50, 100) == AdSize(50, 100));
+  EXPECT_TRUE(AdSize(100, 50) == AdSize(100, 50));
```

Comment:
maybe throw in for good measure...
EXPECT_FALSE(AdSize(50, 100) == AdSize(100, 50))

Reply:
Done!  Added some != tests, too.